### PR TITLE
#17 design and implement location page

### LINKED ISSUE #17: Design and implement location page

### PR DESCRIPTION
**Pull request by erikajansson1**
#### What does this PR do?
Added a Location screen where user can choose his location.
#17   (LINK_TO_ISSUE)

##### Why are we doing this? Any context or related work?
Since we need a user to have a location. The location should later be used in the ad screen to display ads from the users "neighborhood". 
#### Where should a reviewer start?
Location.js
#### Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
closes #17

### CODE REVIEW

**Comment by FredSoderberg** [comment]
Fixed minor stuff in pull request comment

File: Bikeini/screens/Login.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ class Login extends React.Component {
           />
         </View>
 
-        <TouchableHighlight style={[styles.buttonContainer, styles.loginButton]} onPress={this.logInUser}>
+        <TouchableHighlight style={[styles.buttonContainer, styles.loginButton]} onPress={() => navigation.navigate('Location')}>
```
Comment (FredSoderberg) [inline Bikeini/screens/Login.js]: change back to loginUser and set navigate there to location

File: Bikeini/screens/Location.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+};
+
+sendLocationToServer = () => {
+  const { setLocation, loginState } = this.props;
+  serverApi.fetchApi('setLocation', setLocation, 'multipart/form-data', loginState.jwt[0]);
+}
+
+render() {
+  const { checked } = this.state;
+  console.log(this.props);
```
Comment (FredSoderberg) [inline Bikeini/screens/Location.js]: remove log

File: Bikeini/package.json [hunk 3]
```diff
@@ -25,6 +25,11 @@
     "prop-types": "^15.6.2",
     "react": "16.5.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-31.0.0.tar.gz",
+    "react-native-elements": "^0.19.1",
+    "react-native-filesystem": "^0.1.0",
+    "react-native-fs": "^2.12.1",
+    "react-native-radio-buttons": "^1.0.0",
+    "react-native-vector-icons": "^4.6.0",
```
Comment (FredSoderberg) [inline Bikeini/package.json:29]: double check that you still use theese and remove if you dont

File: Bikeini/assets/Cities.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+export const cities = ['Alingsås', 'Arboga', 'Arvika', 'Askersund', 'Avesta', 'Boden', 'Bollnäs', 'Borgholm', 'Borlänge',
```
Comment (FredSoderberg) [inline Bikeini/assets/Cities.js]: NICE!

File: Bikeini/screens/Location.js [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+  return (
+    <View style={styles.container}>
+      <Image style={styles.logo} source={logo} />
+      <Text style={styles.heading}>SELECT CITY</Text>
+      <FlatList
+        style={styles.city}
+        data={cities}
+        keyExtractor={(item, index) => index.toString()}
```
Comment (FredSoderberg) [inline Bikeini/screens/Location.js:96]: neat solution

**Review by FredSoderberg — CHANGES_REQUESTED** [review]
Fix comments pllzzzz 🍡

File: Bikeini/screens/Login.js (hunk 1 above)
Comment (FredSoderberg) [reply Bikeini/screens/Login.js]: i.e the navigate in loginUser()

**Comment by erikajansson1** [comment]
All requested chances are done!

**Review by FredSoderberg — APPROVED** [review]

**Pull request merged**